src/ext.rs

```diff
@@ -130,9 +133,14 @@ pub fn balance(address: &Address) -> U256 {
 pub fn create(endowment: U256, code: &[u8]) -> Result<Address, Error> {
 	let mut endowment_arr = [0u8; 32];
 	endowment.to_big_endian(&mut endowment_arr);
-	let mut result = Address::new();
+	let mut result = Address::from(H160::zero());
 	unsafe {
-		if external::create(endowment_arr.as_ptr(), code.as_ptr(), code.len() as u32, (&mut result).as_mut_ptr()) == 0 {
+		if external::create(
```

Comment:
Just a nit, but below we're using `match`. I don't have a strong preference either way, but I'd prefer it to be consistent.

Reply:
done

Reply:
done